### PR TITLE
Adding Symfony4 support

### PR DESCRIPTION
**Pull request by tattila**
#SymfonyConHackday2017

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -14,6 +14,7 @@ php:
 before_install:
   - phpenv config-rm xdebug.ini || true
   - composer self-update
+  - if [ "$DEPENDENCIES" = "beta" ]; then composer config minimum-stability beta; fi;
```
Comment (sstok) [inline .travis.yml]: You need to update the matrix to include this `DEPENDENCIES` env-variable.

```
matrix:
    include:
        - php: '7.1'
          env: DEPENDENCIES=beta
```

Comment (tattila) [reply .travis.yml]: added

**Comment by Seldaek** [comment]
The 7.1 / sf beta build failed with what seems like real test failures.. Should be looked at before this is mergeable :)

**Comment by tattila** [comment]
i think this problem is fixed:
https://github.com/symfony/symfony/pull/25048

should i retry to commit?

**Comment by Seldaek** [comment]
I triggered a new build, let's see.

**Comment by Seldaek** [comment]
Still fails, I guess it will require a new beta version to work actually..

**Comment by tattila** [comment]
i see, and what should i do now? :)

**Comment by Seldaek** [comment]
I guess.. now we wait for the next beta release :) It's not so urgent

File: composer.json [hunk 2]
```diff
@@ -37,5 +37,6 @@
         "branch-alias": {
             "dev-master": "2.4.x-dev"
         }
-    }
+    },
+    "minimum-stability": "beta"
```
Comment (cordoval) [inline composer.json]: remove this beta, it is now stable

File: .travis.yml [hunk 3]
```diff
@@ -22,3 +23,9 @@ install:
 script:
   - export SYMFONY_PHPUNIT_DIR=`pwd`/.phpunit
   - ./vendor/bin/simple-phpunit
+
+matrix:
+  include:
+    - php: 7.1
+      env: DEPENDENCIES=beta
```
Comment (cordoval) [inline .travis.yml]: remove beta

File: .travis.yml (hunk 1 above)
Comment (cordoval) [reply .travis.yml]: whole line not needed anymore

**Comment by tattila** [comment]
@Seldaek can you trigger a new build?

**Comment by cordoval** [comment]
you can trigger new build by just force pushing on your PR no?

**Comment by tattila** [comment]
with some delay but done @Seldaek @cordoval @sstok

**Comment by Seldaek** [comment]
Cool thanks

**Pull request merged**